recipes/asyncache/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "asyncache" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asyncache-{{ version }}.tar.gz
+  sha256: 9a1e60a75668e794657489bdea6540ee7e3259c483517b934670db7600bf5035
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - pip
+    - poetry-core
+  run:
+    - python >=3.8,<4.0
+    - cachetools >=5.2.0,<6.0.0
+
+test:
+  imports:
+    - asyncache
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: None
```

Pretty sure this is the right home (based on matching user name):
```suggestion
  home: https://github.com/hephex/asyncache
```